### PR TITLE
Fixed group submission bug

### PR DESCRIPTION
**Pull request by JohnPMurray**

### CODE REVIEW

File: backend/config/passport.js [hunk 1]
```diff
@@ -22,7 +22,12 @@ export const samlStrategy = new saml.Strategy({
   // Identity Provider's public key
   cert: samlConfig.cert,
   validateInResponseTo: false,
-  disableRequestedAuthnContext: true
+  disableRequestedAuthnContext: true,
+
+  //forces user to reauthenticate every time they log on
+  // TODO for testing purposes only, remember to remove
+  forceAuthn: true
```
Comment (johnbyrne727rit) [inline backend/config/passport.js:29]: Do we want to make a separate PR for this? so that it's got a heavier paper trail?

File: frontend/src/Student/components/PhotoSubmissionForm.js [hunk 2]
```diff
@@ -161,11 +161,7 @@ class PhotoSubmissionForm extends Component {
     }
 
     // calculate whether the user is beyond their single submissions
-    const numSingleEntries = this.props.data.show.entries.reduce(
-      // if 'student' is non-null, this is a single submission
-      (n, entry) => (entry.student ? n + 1 : n),
-      0
```
Comment (johnbyrne727rit) [inline frontend/src/Student/components/PhotoSubmissionForm.js:167]: OOC What were they trying to do here?

File: backend/config/passport.js (hunk 1 above)
Comment (JohnPMurray) [reply backend/config/passport.js:29]: I feel like the TODO should be enough just to remember we did it. Idk what everyone else thinks

File: frontend/src/Student/components/PhotoSubmissionForm.js (hunk 2 above)
Comment (JohnPMurray) [reply frontend/src/Student/components/PhotoSubmissionForm.js:167]: They were originally basing the disitinction between groups and single submissions based on whether there was an owner of the submission. Now its been changed to check if theres a group in the submission which makes a little more sense.

**Review by johnbyrne727rit — APPROVED** [review]
Looks good to me!

File: backend/config/passport.js (hunk 1 above)
Comment (moiseslorap) [reply backend/config/passport.js:29]: Yeah I feel like this could be something really easy to lose track of down the line. I wouldn't think necessarily a pull request. Maybe have it in writing as an issue to address later on!

**Review by moiseslorap — APPROVED** [review]
